pkg/app/mongodb-deploymentconfig.go

```diff
@@ -34,31 +34,31 @@ import (
 
 const (
 	mongoDepConfigName        = "mongodb"
-	mongoDepConfigWaitTimeout = 4 * time.Minute
+	mongoDepConfigWaitTimeout = 5 * time.Minute
 )
 
 type MongoDBDepConfig struct {
-	cli        kubernetes.Interface
-	osCli      osversioned.Interface
-	name       string
-	namespace  string
-	dbTemplate string
-	user       string
-	label      string
-	osClient   openshift.OSClient
-	params     map[string]string
+	cli         kubernetes.Interface
+	osCli       osversioned.Interface
+	name        string
+	namespace   string
+	user        string
+	label       string
+	osClient    openshift.OSClient
+	params      map[string]string
+	storageType storage
 }
 
 func NewMongoDBDepConfig(name string) App {
```

Comment:
Should we accept storagetype as a argument to the function? It will allow use to switch storage types if we import and use this in another project.